api/turing/middleware/authorization.go

```diff
@@ -85,7 +85,7 @@ func (a *Authorizer) FilterAuthorizedProjects(
 		return nil, err
 	}
 
-	var allowedProjects []mlp.Project
+	allowedProjects := make([]mlp.Project, 0)
```

Comment:
Otherwise `allowedProjects` could be `nil`, if  `allowedProjectIds` is empty

Reply:
They are actually equavilent. `make([]mlp.Project, 0)` == `var allowedProjects []mlp.Project`. Can try here: https://play.golang.org/p/BZYWfhQnXO4

Reply:
They aren't. Check nil slice vs empty slice in Golang, for example: https://medium.com/@habibridho/golang-nil-vs-empty-slice-87fd51c0a4d

Reply:
What's important, that: 
```go
fmt.Println(nilSlice == nil) // Output: true
fmt.Println(emptySlice == nil) // Output: false
```
So when you try to marshal empty slice into JSON, you'll get `[]`, but marshalling of nil slice will end up with `null`

Reply:
Ah interesting.